README.md

```diff
@@ -77,7 +77,7 @@ You can find some tips and tricks about the ReSharper.Host plugin (the *backend*
 
 ## Development notes
 
-The main development branch is changed on each release (i.e. `net212` correspond to 2021.2 development cycle), and builds from this branch are bundled with nightly Rider builds available via [JetBrains Toolbox App](https://www.jetbrains.com/toolbox/app/).
+The `main` branch is used for development, and builds from this branch are bundled with nightly Rider builds available via [JetBrains Toolbox App](https://www.jetbrains.com/toolbox/app/).
```

Comment:
The second part becomes incorrect with this change. 🙂

Internally, there're still branches like `243` and `251`, and this is where nightly build are built. We can probably remove this part for now.

Reply:
Updated. Thanks for the review